scripts/create_text_dump_of_source_data.py

```diff
@@ -54,19 +68,35 @@ def process_dictionary(dictionary: typing.Dict) -> str:
 def main(ctx, db_url, verbosity, heart_rate, authority):
     isb_lib.core.things_main(ctx, db_url, None, verbosity, heart_rate)
     session = SQLModelDAO((ctx.obj["db_url"])).get_session()
-    sql = f"""select resolved_content from thing tablesample system(1) where
+    sql = f"""select resolved_content from thing tablesample system(10) where
     resolved_status=200 and authority_id='{authority}'"""
     rs = session.execute(sql)
     filename = f"{authority}.txt"
-    lines = []
+    result_set_dicts = []
+    header_fieldnames = set()
     for row in rs:
         resolved_content = row._asdict()["resolved_content"]
-        row_string = process_dictionary(resolved_content)
-        row_string = row_string + "\n"
-        print("row_string is " + row_string)
-        lines.append(row_string)
-    with open(filename, "w") as file:
-        file.writelines(lines)
+        processed_resolved_content = {}
+        process_dictionary(
+            None, header_fieldnames, resolved_content, processed_resolved_content
+        )
+        result_set_dicts.append(processed_resolved_content)
+
+    with open(filename, "w", newline="") as file:
+        writer = csv.DictWriter(
+            file,
+            delimiter="#",
+            quoting=csv.QUOTE_ALL,
+            fieldnames=list(header_fieldnames),
+        )
+        writer.writeheader()
+        for result_set_dict in result_set_dicts:
+            writer.writerow(result_set_dict)
+
+    with ZipFile(
+        filename + ".zip", "w", compression=zipfile.ZIP_DEFLATED
+    ) as out_zipfile:
+        out_zipfile.write(filename)
```

Comment:
The SESAR file was 634MB zipped, so go ahead and zip everything (zipped it went down to 35MB).